### PR TITLE
use expose loader instead of writing to window directly

### PR DESCRIPTION
**Pull request by k-nut**

### CODE REVIEW

File: theme/src/scripts/index.js [hunk 1]
```diff
@@ -8,4 +8,5 @@ import 'babel-polyfill'
 
 // style
 import "../styles/main.sass";
+require("expose-loader?charts!./charts.js")
```
Comment (sjockers) [inline theme/src/scripts/index.js]: Is there a special reason for using `require` over `import`? They should do the same in this context and I'd to stick with import statements for consistency.

File: theme/package.json [hunk 2]
```diff
@@ -47,7 +47,7 @@
     "eslint-loader": "^1.3.0",
     "extract-text-webpack-plugin": "^1.0.1",
     "html-webpack-plugin": "^2.17.0",
-    "node-sass": "^4.0.0",
+    "node-sass": "^4.9.0",
```
Comment (sjockers) [inline theme/package.json]: This is not related, right?

Comment (k-nut) [reply theme/package.json]: good catch!

File: theme/src/scripts/index.js (hunk 1 above)
Comment (k-nut) [reply theme/src/scripts/index.js]: Ah nice, I thought the doucmentation said that this only works with `require` but apparently it also works with `import` which I also like better for consistency :)

**Comment by k-nut** [comment]
@sjockers changed and force pushed.
